workshop_3/workshop_3.ino

```diff
@@ -0,0 +1,56 @@
+#include <Adafruit_Sensor.h> // include these three libraries (Adafruit_Sensor.h, DHT.h, DHT_U.h)
+#include <DHT.h> 
+#include <DHT_U.h>
+DHT_Unified dht(2, DHT11);   // to define that sensor DHT11 is on pin 2
```

Comment:
Please insted of hardcoded `2` use a define

Reply:
comment ça?